fenix/app/src/main/java/org/mozilla/fenix/translations/preferences/automatic/PrefAutomaticTranslation.kt

```diff
@@ -0,0 +1,117 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.translations.preferences.automatic
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.semantics.heading
+import androidx.compose.ui.semantics.semantics
+import androidx.compose.ui.unit.dp
+import org.mozilla.fenix.R
+import org.mozilla.fenix.compose.annotation.LightDarkPreview
+import org.mozilla.fenix.compose.list.TextListItem
+import org.mozilla.fenix.theme.FirefoxTheme
+import java.util.Locale
+
+/**
+ * Firefox Preferences Automatic Translate fragment compose view.
+ *
+ * @param prefAutomaticTranslationList List of [PrefAutomaticTranslationItem]s to display.
+ * @param onItemClick Invoked when the user clicks on the a item from the list.
+ */
+@Composable
+fun PrefAutomaticTranslation(
```

Comment:
```suggestion
fun AutomaticTranslationPreference(
```
Gonna suggest a different way of naming all of these preferences. My thinking is that we would rather know upfront what the title of the screen says. wdyt?

Reply:
I made one of these suggestions [originally](https://github.com/mozilla-mobile/firefox-android/pull/4226#discussion_r1399554662). The other patch will need to be updated too, if we change directions on conventions.

My thoughts with keeping the prefix are that we are going to end up with a lot of preferences on this feature and it would be nice to have a way to categorize them as distinct from other elements. Open to whatever you think!

Reply:
#4356 is the other bug we will need to comment on, if we move to a different convention.

Reply:
@ohall-m I like the idea of grouping these things into folders as you suggested with `translations/preferences/automatic` in your original suggestion. I think I am also gravitating towards having `Pref` or `Preference` as the suffix instead of prefix.

Reply:
Sounds good to me!